File: URI/Python/Quadrante.py
while True:
    inf = [int(a) for a in input().split()]
    x = inf[0]
    y = inf[1]
    if x > 0 and y > 0:
        print("primeiro")
    elif x < 0 and y > 0:
        print("segundo")
    elif x < 0 and y < 0:
        print("terceiro")
    elif x > 0 and y < 0:
        print("quarto")

    if x == 0 or y == 0:
        break

File: URI/Lua/Quadrante.lua
while 1 do
    x = io.read("*number")
    y = io.read("*number")
    if x > 0 and y > 0 then
        print("primeiro")
    end
    if x < 0 and y > 0 then
        print("segundo")
    end
    if x < 0 and y < 0 then
        print("terceiro")
    end
        if x > 0 and y < 0 then
        print("quarto")
    end

    if x == 0 or y == 0 then
        break
    end
end